airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -66,12 +64,14 @@ def __init__(self, conn_id='sqoop_default', verbose=False,
         self.hcatalog_table = hcatalog_table
         self.verbose = verbose
         self.num_mappers = num_mappers
-        self.properties = properties
+        self.properties = properties if properties else {}
+        self.log.info("Using connection to: {}:{}/{}".format(self.conn.host, self.conn.port, self.conn.schema))
```

Comment:
The connection log message just shows the host but it'd be good to know the full connection Sqoop is trying to connect to.